www/src/components/sponsors/data.ts

```diff
@@ -0,0 +1,204 @@
+export const sponsors = [
```

Comment:
hardcoded data until we replace with job.

I didn't fully verify `individuals` in case you wanted to take this over and have the data be fed from the source. If we want to hold on that I can manually verify the whole list